apps/api/src/app/environments/environment.ts

```diff
@@ -103,6 +103,18 @@ const devConfig = {
       process.env.XROAD_TJODSKRA_API_PATH ?? '/SKRA-Protected/Einstaklingar-v1',
     clientId: process.env.XROAD_CLIENT_ID ?? 'DEV/GOV/10000/island-is-client',
   },
+  paymentDomain: {
+    xRoadBaseUrl: process.env.XROAD_BASE_PATH,
+    xRoadProviderId:
+      process.env.PAYMENT_XROAD_PROVIDER_ID ??
+      'IS-DEV/GOV/10021/FJS-DEV-Public',
+    xRoadClientId: process.env.XROAD_CLIENT_ID,
+    username: process.env.PAYMENT_USER,
+    password: process.env.PAYMENT_PASSWORD,
+    callbackBaseUrl: process.env.PAYMENT_BASE_CALLBACK_URL,
+    callbackAdditionUrl: process.env.PAYMENT_ADDITION_CALLBACK_URL,
+    arkBaseUrl: 'https://uat.arkid.is',
```

Comment:
Please create a secret for this: https://islandis.slack.com/archives/C013A6RBD34/p1623158608035800?thread_ts=1623064816.032800&cid=C013A6RBD34

Reply:
@sindrig great reminder. The tests are crucial to keep the quality to standards.